front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js

```diff
@@ -35,12 +35,14 @@ export default function DriverTrainSchedule(props) {
     if (data) getRollingStock();
   }, [data]);
 
-  return (
+  return isModal ? (
     <ModalSNCF className="modal-drivertrainschedule" htmlID="driverTrainScheduleModal" size="xl">
       <ModalBodySNCF>
         {data && rollingStockSelected ? (
```

Comment:
if the second expression is empty, no need for a ternary, you can simplify
`data && rollingStockSelected && <Component />`

Reply:
It's not really empty, <ModalBodySNCF /> needs a children, so '' is not nothing and it stops complaining :)